.github/workflows/docker_builder.yml

```diff
@@ -0,0 +1,44 @@
+name: 'Torch-TRT Docker Build'
+
+# Apply workflow only to main branch
+on:
+  push:
+    branches: [ main ]
+
+# If pushes to main are made in rapid succession,
+# cancel existing docker builds and use newer commits
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: linux.2xlarge
+
+    # Define key environment variables
+    env:
+      DOCKER_REGISTRY: ghcr.io/pytorch/tensorrt
+      CONTAINER_NAME: torch_tensorrt_main:latest
```

Comment:
Current container naming is `torch_tensorrt_main:latest`. Some alternatives are:

- `torch_tensorrt:main`
- `torch_tensorrt:latest`
- `torch_tensorrt:COMMIT_SHA` (could cause registry size issues since each push would have a unique tag)
- `torch_tensorrt:devX.Y.Z`